packages/api-client/src/views/Request/RequestSubpageHeader.vue

```diff
@@ -20,14 +21,19 @@ defineEmits<{
 }>()
 
 const { activeCollection } = useWorkspace()
+const { layout } = useLayout()
 </script>
 <template>
   <div
     class="lg:min-h-header flex items-center w-full justify-center p-2 pt-2 lg:pt-1 lg:p-1 flex-wrap t-app__top-container border-b-1/2">
     <div
       class="flex flex-row items-center gap-1 lg:px-1 lg:mb-0 lg:mb-0 mb-2 lg:flex-1 w-6/12">
       <SidebarToggle
-        class="gitbook-hidden"
+        :class="[
```

Comment:
I think maybe we can avoid the importants here?

```suggestion
        class="gitbook-hidden"
        :class="[
        layout === 'modal' : 'flex' :
          modelValue ? 'xl:hidden' : 'xl:flex'
        ]"
```